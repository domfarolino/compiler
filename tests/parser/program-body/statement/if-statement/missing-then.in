program dominic is

  integer dominic;

begin
  dominic := 10;

  if (dominic == 10)
    dominic := dominic + 1;
  end if;

end program.
