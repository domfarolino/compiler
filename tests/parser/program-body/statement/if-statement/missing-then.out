[1;31mLine 9: Expected 'then' after '( <expression> )' in if statement, but got: 'dominic'
Line 9: Error parsing program body
[0m